### PR TITLE
Patch/create task

### PR DESCRIPTION
**Pull request by Paul-Vasylenko**
Completely ready create-task page.
Ready for QA

### CODE REVIEW

**Comment by Paul-Vasylenko** [comment]
[[Create Task] make "Your challenge", "Go to" Selects work properly](https://trello.com/c/9AcCcFyM/133-create-task-make-your-challenge-go-to-selects-work-properly)

**Comment by Paul-Vasylenko** [comment]
[fix full screen in create task page](https://trello.com/c/su9c6I63/108-fix-full-screen-in-create-task-page)

**Review by nikita-dnestrov — APPROVED** [review]
Haven't noticed anything strange. Double check before merging just in case

File: client/src/components/pages/create-task/common/create-tabs/create-tabs.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Header } from './header';
 import { IconTaskPageFullScreen } from 'common';
 import { TabContent } from './tab-content';
 import { ICreateTabsProps } from './types';
-
+import './styles.scss';
```
Comment (AlexanderKhapchenko) [inline client/src/components/pages/create-task/common/create-tabs/create-tabs.tsx:8]: why without "module"?

File: client/src/containers/user/logic/action-types.ts [hunk 2]
```diff
@@ -12,3 +15,11 @@ export type TSetUser = {
...
 	accessToken: IUserDataState['accessToken'];
 };
+
+export type TUserAddTask = {
+	task: WebApi.Entities.ITask
+};
+
+export type TUserDeleteTask = {
+	task: WebApi.Entities.ITask
+};
```
Comment (AlexanderKhapchenko) [inline client/src/containers/user/logic/action-types.ts:25]: EOL

File: client/src/enum/api/task-api-path.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
 export const TaskApiPath = {
 	SEARCH: 'tasks/search',
+	ROOT: '/tasks/'
```
Comment (AlexanderKhapchenko) [inline client/src/enum/api/task-api-path.ts:3]: Why is the search without a slash at the beginning of the word?
And the root has a slash at the beginning of the word

File: client/src/typings/common/IUser.ts [hunk 4]
```diff
@@ -4,12 +4,13 @@ import { IGithubProfile } from 'typings/common/IGithub';
...
-	rank: number;
-	honor: number;
+	rank: number; //where is rank? should be 8 by default
+	honor: number;  //where is honor?
 	name: string;
 	img?: string;
 	surname?: string;
 	clan?: WebApi.Entities.IClan;
-	profileUrl?: string;
+	profileUrl?: string; //?
```
Comment (AlexanderKhapchenko) [inline client/src/typings/common/IUser.ts:13]: Is this commentary needed here?

File: client/src/typings/common/IUser.ts [hunk 5]
```diff
@@ -4,12 +4,13 @@ import { IGithubProfile } from 'typings/common/IGithub';
 export interface IUser {
 	id: string;
 	username: string;
-	rank: number;
-	honor: number;
+	rank: number; //where is rank? should be 8 by default
+	honor: number;  //where is honor?
```
Comment (AlexanderKhapchenko) [inline client/src/typings/common/IUser.ts:8]: Is this commentary needed here?

**Pull request merged**